### PR TITLE
Fixes #35712 - CV version compare UX reviews

### PR DESCRIPTION
**Pull request by sjha4**
#### What are the changes introduced in this pull request?
Screenshot 2022-11-03 at 15 26 58 (1)

#### What are the testing steps for this pull request?
Screenshot from 2022-11-09 16-53-23
Screenshot from 2022-11-09 16-53-00

### CODE REVIEW

File: webpack/scenes/ContentViews/Details/Versions/Compare/CVVersionCompareTable.js [hunk 1]
```diff
@@ -78,8 +79,15 @@ const CVVersionCompareTable = ({
...
             <Tr key={`column-${result.id}`} ouiaId={`column-${result.id}`}>
               {columnHeaders.map(({ getProperty }, colIndex) =>
-                // eslint-disable-next-line react/no-array-index-key
-                <Td key={`cell-${colIndex}`}>{getProperty(result)} </Td>)}
+                (
+                  <Td
+                    className={(colIndex === columnHeaders.length - 3) ? 'border-right' : ''}
+                  // eslint-disable-next-line react/no-array-index-key
+                    key={`cell-${colIndex}`}
+                    style={(colIndex >= columnHeaders.length - 2) ? { paddingLeft: '2rem' } : {}}
```
Comment (jeremylenz) [inline webpack/scenes/ContentViews/Details/Versions/Compare/CVVersionCompareTable.js:87]: Stack overflow says this has a CSS equivalent.. 
```css
td:nth-last-child(-n+2)
```

**Review by jeremylenz — COMMENTED** [review]
image

File: webpack/scenes/ContentViews/Details/Versions/Compare/CVVersionCompareTable.js (hunk 1 above)
Comment (sjha4) [reply webpack/scenes/ContentViews/Details/Versions/Compare/CVVersionCompareTable.js]: Updated to use td:nth-last-child for the borders.

**Review by jeremylenz — APPROVED** [review]
Looking great!

Ack :+1:

**Pull request merged**